xtend-website/_posts/releasenotes/2020-03-03-version-2-21-0.md

```diff
@@ -6,7 +6,11 @@ categories: releasenotes
 published: false
 ---
 
-Xtend 2.21.0 is mainly a maintenance release with a small number of usability improvements
+Xtend 2.21.0 is mainly a maintenance release with a small number of usability improvements.
```

Comment:
.. is a maintenance release with a number of ...

Reply:
we.. did not change that => should be updated separately

Reply:
ok. Just caught my attention.